templates/ade_corpus_v2/Ade_corpus_v2_classification/templates.yaml

```diff
@@ -1,9 +1,43 @@
 dataset: ade_corpus_v2
 subset: Ade_corpus_v2_classification
 templates:
-  7670c23f-d76e-4491-bad3-55ea6b0073f7: !Template
-    id: 7670c23f-d76e-4491-bad3-55ea6b0073f7
-    jinja: '{{text}} Is this sentence Adverse Drug Event {{"related or not"}}?|||
-      {{["Not-Related", "Related"][label]}}'
-    name: Test1
+  16830a03-568f-4374-9c77-263d8ab50d71: !Template
+    id: 16830a03-568f-4374-9c77-263d8ab50d71
+    jinja: "Read the below text and answer the question.\n\nText: {{text}} \n\nQuestion:\
+      \ Is the text related to adverse drug effect (ADE)?\n\n1. Yes\n2. No\n|||\n\
+      {{[\"Yes\", \"No\"][label]}}"
+    name: baseline-even-more-verbose
+    reference: ''
+  469d2f5a-9eb0-4a05-8110-231016c36068: !Template
+    id: 469d2f5a-9eb0-4a05-8110-231016c36068
+    jinja: "Text: {{text}} \n\nQuestion: Is the text related to adverse drug effect\
+      \ (ADE)?\n|||\n{{[\"Not-Related\", \"Related\"][label]}}"
+    name: baseline-verbose
+    reference: ''
+  5398b646-fc60-4483-915c-9d1f0136412c: !Template
+    id: 5398b646-fc60-4483-915c-9d1f0136412c
+    jinja: Is "{{text}}" related to adverse drug effect (ADE)? ||| {{["Yes", "No"][label]}}
+    name: baseline-YN
+    reference: ''
+  56bd12a8-b8ee-464e-98cc-5f586ba9f74d: !Template
+    id: 56bd12a8-b8ee-464e-98cc-5f586ba9f74d
+    jinja: Is "{{text}}" related to adverse drug effect (ADE)? ||| {{["Not-Related",
```

Comment:
You can add `text.rstrip(".")` to your templates since I am under the impression that "text" is always ending with "."
I think that's fine if you are not adding these everywhere (we want this type of noise)

Reply:
I didn't add this everywhere but I'll keep in mind for sure.